### PR TITLE
Add megastudy presence

### PR DESCRIPTION
**Pull request by 1LUXION**
> **megastudy** is the largest Internet lecture service in South Korea.
> I made the presence of megastudy in order to motivate students of Korea,
> so it may be difficult to review my code since it is made for Korean Site and Korean Students.
> I'm sorry for the inconvenience, and I appreciate reviewers' hard work.

### Home Screen
home

###  Lecture Room
lecture

###  Lecture Paused
lecture paused

###  Lecture Playing
lecture playing

### CODE REVIEW

**Review by Slowlife01 — COMMENTED** [review]
Add support for all sub-pages?

File: websites/M/megastudy/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "megastudy",
```
Comment (Bas950) [inline websites/M/megastudy/dist/metadata.json:3]:
```suggestion
  "service": "megastudy",
  "altnames": ["메가스터디"],
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issues

**Comment by 1LUXION** [comment]
> Fix codacy issues

@Bas950 all fixed, thanks for adding alternative service name

**Comment by 1LUXION** [comment]
> Add support for all sub-pages?

@SlowLife1661 Added support for most of sub pages as long as I can, but its technically hard to add all sub pages since megastudy changes Sub Page Structure every time they recruit a new teacher or offer a new course.

**Review by Slowlife01 — DISMISSED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Set browsingstamp once at the start, instead of in every if case.

**Comment by 1LUXION** [comment]
> Set browsingstamp once at the start, instead of in every if case.

@Bas950 Simplified all statements and assignments

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**